src/jit_runner.cpp

```
@@ -0,0 +1,374 @@
+                if (!canonical_share.empty()) {
+                    auto canonical = fs::weakly_canonical(fname).string();
+                    is_stdlib = canonical.find(canonical_share) == 0;
+                }
+                cs->filenames[gid] = is_stdlib ? "" : fname;
+            }
+            cs->file_count = new_total;
+        }
+        cs->file_id_offset += fc;
+    }
+
+    return result > 0 ? 1 : 0;
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)